core/src/main/scala/lgbt/princess/lifts/LiftKind.scala

```diff
@@ -0,0 +1,94 @@
+package lgbt.princess.lifts
+
+import cats.arrow.FunctionK
+import cats.data.{EitherT, IorT, Kleisli, OptionT, StateT, WriterT}
+import cats.{Applicative, Functor, Monoid, ~>}
+
+import scala.annotation.implicitNotFound
+
+/**
+ * Lifts values and scope transformations from the higher-kinded type `F` to the higher-kinded type
+ * `G`.
+ *
+ * TODO: document issue with `StateT` implicit
+ */
+@implicitNotFound("no way defined to lift values and scopes from ${F} to ${G}")
+trait LiftKind[F[_], G[_]] extends LiftValue[F, G] with LiftScope[F, G] {
+  def andThen[H[_]](that: LiftKind[G, H]): LiftKind[F, H] =
+    new LiftKind.Composed(this, that)
+
+  def compose[E[_]](that: LiftKind[E, F]): LiftKind[E, G] =
+    new LiftKind.Composed(that, this)
+}
+
+object LiftKind {
+
+  private final class Composed[F[_], G[_], H[_]](inner: LiftKind[F, G], outer: LiftKind[G, H])
+      extends LiftKind[F, H] {
+    def liftF[A](value: F[A]): H[A] = outer.liftF(inner.liftF(value))
+    val liftK: F ~> H = inner.liftK.andThen(outer.liftK)
+    def liftScopeApply[A](scope: F ~> F)(value: H[A]): H[A] =
+      outer.liftScopeApply(inner.liftScope(scope))(value)
+    override def liftScope(scope: F ~> F): H ~> H =
+      outer.liftScope(inner.liftScope(scope))
```

Comment:
I expected this not to be lawful. I expected it to run the function twice (I struggle to wrap my mind around the nesting). but it passes the tests with flying colours